src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -15,21 +15,19 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Definitions
 
-(def jruby-puppet-env
-  "The environment variables that should be passed to the Puppet JRuby interpreters.
-  We don't want them to read any ruby environment variables, like $GEM_HOME or
-  $RUBY_LIB or anything like that, so pass it an empty environment map - except -
-  Puppet needs HOME and PATH for facter resolution, so leave those."
-  (select-keys (System/getenv) ["HOME" "PATH"]))
-
 (def ruby-code-dir
-  "The name of the directory containing the ruby code in this project.
-  This directory lives under src/ruby/"
-  "puppet-server-lib")
+  "The name of the directory containing the ruby code in this project."
+  "src/ruby/puppet-server-lib")
```

Comment:
This is the path into the ruby code from the root of the `puppet-server-release.jar`.  

@jeffmccune I bet if you built packages with lein ezbake with this change that the CLI tools might not work.